assets/templates/authorize.html

```diff
@@ -40,9 +40,9 @@
                 <strong>{{.Domain}}</strong> :<br />
                 {{end}}
               </p>
-              <ul>
+              <ul class="perm-list">
                 {{range $index, $perm := .Permissions}}
-                <li>
+                <li class={{ $perm.Type }}>
```

Comment:
oh, right. I'm used to not put quotes in JS but yeah it think it should with Go. Browsers apparently add them automatically, but you're right, just to be sure i'll add them.